lasagne/layers/recurrent.py

```diff
@@ -356,7 +364,10 @@ def get_output_for(self, inputs, **kwargs):
             part of the sequence or not.  `mask` should be a matrix of shape
             ``(n_batch, n_time_steps)`` where ``mask[i, j] = 1`` when ``j <=
             (length of sequence i)`` and ``mask[i, j] = 0`` when ``j > (length
-            of sequence i)``.
+            of sequence i)``. When the hidden state of this layer is to be
+            pre-filled (i.e. was instantiated with `isinstance(hid_init,
+            Layer)`) `inputs` should have length at least 2, and `inputs[-1]`
```

Comment:
I think `== True` can be omitted, it doesn't change that expression's value. Double backticks would be correct, yes. Or maybe reformulate as
```
When the hidden state of this layer is to be
pre-filled (i.e., `hid_init` was set to a
:class:`Layer` instance), ...
```